### ISSUE TITLE
Do not break already-installed packages in other directories

### ISSUE THREAD

**Issue by davidedelvento** [pradyunsg/zazo#17]
Following the suggestion in https://github.com/pypa/pip/issues/988#issuecomment-608040274 I am submitting this test case here.

Instead of just repeating myself, the issue is https://github.com/pypa/pip/issues/6494 (and some additional info are in the linked tickets from there).

Not sure if that is in scope with the new dependency resolver, but mentioning it and let you think about it for a moment can't hurt.

**Comment by pradyunsg** [comment]
Looks like you submitted the same issue twice.

**Comment by davidedelvento** [comment]
No, these are two different things. This one is about installing in "non default" locations with the `-t` option, which is the less serious of the problem since it affects only people who cannot use virtual environments. The other is most serious and I'll comment there what it is. You might want to reopen it.

**Comment by pradyunsg** [comment]
Hi, sorry about the confusion here. The titles of both those requests were very similar and I acted a bit less carefully than I should've. I took the liberty to re-title these issues to remove the duplication.

I see that there's relevant discussion about this in https://github.com/pypa/pip/issues/5178 as well.

**Comment by davidedelvento** [comment]
No need to be sorry. You have quite a lot to track and need to be quick to find the time to actually do stuff rather than just clerical work on the issue tracker. Changing anything to make it clearer is good.

Thanks!
